Modules/Common/AptosLikeMainModule.php

```diff
@@ -0,0 +1,297 @@
+<?php declare(strict_types=1);
+
+/*  Idea (c) 2023 Nikita Zhavoronkov, nikzh@nikzh.com
+ *  Copyright (c) 2023 3xpl developers, 3@3xpl.com, see CONTRIBUTORS.md
+ *  Distributed under the MIT software license, see LICENSE.md  */
+
+/*  This module process the APT transfers in Aptos Blockchain. */
+
+abstract class AptosLikeMainModule extends CoreModule
+{
+    use AptosTraits;
+
+    public ?BlockHashFormat $block_hash_format = BlockHashFormat::HexWith0x;
+    public ?AddressFormat $address_format = AddressFormat::HexWith0x;
+    public ?TransactionHashFormat $transaction_hash_format = TransactionHashFormat::HexWith0x;
+    public ?TransactionRenderModel $transaction_render_model = TransactionRenderModel::Even;
+    public ?CurrencyFormat $currency_format = CurrencyFormat::Static;
+    public ?CurrencyType $currency_type = CurrencyType::FT;
+    public ?FeeRenderModel $fee_render_model = FeeRenderModel::ExtraF;
+    public ?array $special_addresses = ['the-void', 'the-contract'];
+    public ?PrivacyModel $privacy_model = PrivacyModel::Transparent;
+    public ?bool $ignore_sum_of_all_effects = false;
```

Comment:
Same as above

Reply:
There is no such case, so the flag is set to `false`.